frontend/src/metabase/user/components/LoginHistoryList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+
+import LoginHistory from "metabase/entities/loginHistory";
+
+const LoginHistoryItemRow = item => {
+  return (
+    <tr>
+      <td>
+        <tt>{item.timestamp}</tt>
+      </td>
+      <td>{item.location && item.location.description}</td>
```

Comment:
What are the possible formats for location? It'd be nice if we could somehow trim down the string to "San Francisco, CA" or "Brooklyn, NY" where possible since the other stuff creates noise and actually makes it less likely to be able to spot a login that doesn't fit.